currency_rate_update/services/update_service_GOOGLE.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (c) 2009 CamptoCamp. All rights reserved.
+#    @author Nicolas Bessi
+#    @author Ross Golder
+#
+#    Abstract class to fetch rates from Google Currency Converter
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```

Comment:
Can you use a shorter header ?
e.g. https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py